utils/exr_utils.py

```diff
@@ -157,10 +157,23 @@ def reject_outliers(data, m=2):
     return np.all(s < m, axis=1)
 
 
+<<<<<<< HEAD
```

Comment:
Please clean up this merge conflict ;) 
it should be:
``` python
def exr_2_numpy(exr_file: str) -> np.ndarray:
    """
    read an exr file using cv2.
    Note: if the file is depth from blender, it could be the same value repeated 3 times (rgb).

    :param exr_file: path to the file
    :return: the cv2 imported image
    """
    return cv2.imread(exr_file, cv2.IMREAD_ANYCOLOR | cv2.IMREAD_ANYDEPTH)
```